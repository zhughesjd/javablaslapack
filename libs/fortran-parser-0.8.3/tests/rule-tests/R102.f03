

! Check for generic name list. Using derived type statement.
type coolguy (this,that,theother)
	integer age
end type coolguy
end
