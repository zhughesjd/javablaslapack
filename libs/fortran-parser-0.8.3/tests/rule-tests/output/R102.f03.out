1101:begin
102:list-begin
102:part this
102:part that
102:part theother
102:list 3
430 coolguy
403 INTEGER false
442:list-begin
442:list 1
430 coolguy
1103
1101 false false false
